examples/server_profile.rb

```diff
@@ -111,15 +120,15 @@
   )
   ethernet_network_connections [
     {
-      EthernetNetwork1: {
+      mgmt_untagged: {
```

Comment:
change this name to 'iscsi_nw'.
There will be 2 connections with iscsi_nw. One with primary boot and other with secondary boot, both uses same network.